src/components/layout/ScreenContentFrame.tsx

```diff
@@ -10,9 +11,11 @@ interface FrameProps {
 
 export function ScreenContentFrame(props: PropsWithChildren<FrameProps>) {
   const { onClose } = props
+  const warnings = useHomeScreenWarnings()
+  const height = warnings ? { height: 'calc(100% - 5em)' } : { height: '100%' }
```

Comment:
This approach is okay but to avoid needing warning layout logic in two places, could we instead have ScreenFrameWithFeed wrap the children content in a container who's max height is computed there? That way this can still assume 100%. My main concern is that if ScreenFrameWithFeed has other children than ScreenContentFrame, those other children will need this logic too.

Reply:
Ok, after taking another look, I was actually able to add it to Home Screen (when not Mobile) so it's within the content frame.  This eliminates the need for the layout logic in ScreenContentFrame, and also removes the need for the hook completely.  See if you think this looks better.